### PR TITLE
Add a package definition for althea-dash

### PR DESCRIPTION
**Pull request by drozdziak1**
**Note:** Please don't merge, I left a reference to my fork (instead of althea-mesh/*) in here so that testing the firmware PR (listed below on the timeline) is a bit easier. I'll force push and merge once the review comes through.

### CODE REVIEW

**Comment by jkilpatr** [comment]
@drozdziak1 we need both yarn and npm in the deps list for the althea-firmware pull in pre-install-setup

**Comment by drozdziak1** [comment]
@jkilpatr Not really, I probably left it because I wasn't sure how `npm` and `yarn` worked together.

File: althea/althea-dash/Makefile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+define Package/althea-dash/description
+  UI for controlling the behavior of Althea from your web browser
+endef
+
+define Build/Compile
+	( \
+	cd $(PKG_BUILD_DIR) &&\
+	npm install &&\
+	npm run build \
```
Comment (drozdziak1) [inline althea/althea-dash/Makefile]: FYI Not using `yarn` for this anymore.

File: althea/althea-dash/Makefile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=althea-dash
+PKG_VERSION:=0.1
+PKG_RELEASE:=0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/althea-mesh/althea-dash
```
Comment (drozdziak1) [inline althea/althea-dash/Makefile]: URL changed back to upstream

**Pull request merged**